requirements-dev.txt

```diff
@@ -1,3 +1,5 @@
 black==20.8b1
 isort==5.6.4
 pre-commit==2.9.3
+aiotruenas-client===0.4.0
```

Comment:
In an ideal world, we'd have a test to verify that this and the HACS json specify the same version.  You don't need to fix that here, but can you file an issue on it please?